templates/index.j2

```diff
@@ -74,7 +74,7 @@
         const discordLinkInput = document.getElementById("discordLink");
         const githookLinkOutput = document.getElementById("githookLink");
 
-        const webhook_link_regex = /https:\/\/(?:canary\.)?discordapp\.com\/api\/webhooks\/(\d+)\/([\w-]+)/;
+        const webhook_link_regex = /https:\/\/(?:canary\.)?discord\.com\/api\/webhooks\/(\d+)\/([\w-]+)/;
```

Comment:
You should add an optional `(app)?` matcher as well, to allow people to use discordapp.com as well as discord.com.
`discord\.com` -> `discord(app)?\.com`.

Reply:
https://canary.discord.com/channels/135634590575493120/804050388680310784/912505289956294766